src/libsystemd-network/network-common.c

```diff
@@ -4,21 +4,25 @@
 #include "network-common.h"
 #include "string-util.h"
 
-const char *get_ifname(int ifindex, char **ifname) {
-        char buf[IF_NAMESIZE + 1];
+int get_ifname(int ifindex, char **ifname) {
```

Comment:
rename second arg `ret_ifname` maybe?

Reply:
maybe rename this format_ifname_malloc() and move it to generic code

Reply:
The function do nothing when `*ifname` is set. We usually use `ret` or `ret_` prefix when the argument is purely used for storing results.